using-mobile-extensions/adobe-analytics-mobile-services/README.md

```diff
@@ -738,24 +751,37 @@ MobileServices.trackAdobeDeepLink
 **Swift**
 
 ```swift
-- (BOOL)application:(UIApplication *)application handleOpenURL:(NSURL *)url {
-    [AEPMobileServices trackAdobeDeepLink:url];
+func application(_ application: UIApplication, handleOpen url: URL) -> Bool {
```

Comment:
this API seems to be deprecated https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622964-application
Can we keep only the example for _:open:options: below?

Reply:
I'm okay with that. It looks like the API is available in iOS 2-9, which we don't support. The Assurance API docs don't include this API either. @sbenedicadb are you okay with removing this API from the samples was well?